src/edu/ucsb/nceas/metacat/dataone/MNodeService.java

```diff
@@ -479,14 +479,27 @@ public Identifier update(Session session, Identifier pid, InputStream object,
         String invalidRequestCode = "1202";
         String notFoundCode ="1280";
         SystemMetadata existingSysMeta = getSystemMetadataForPID(pid, serviceFailureCode, invalidRequestCode, notFoundCode, true);
+        D1AuthHelper authDel = null;
         try {
-            D1AuthHelper authDel = new D1AuthHelper(request,pid,"1200","1310");
-            authDel.doUpdateAuth(session, existingSysMeta, Permission.WRITE, this.getCurrentNodeId());
+            authDel = new D1AuthHelper(request,pid,"1200","1310");
+            authDel.doUpdateAuth(session, existingSysMeta, Permission.CHANGE_PERMISSION, this.getCurrentNodeId());//if the user has the change permission, it will be all set; otherwise, we need to check more.
```

Comment:
```suggestion
            //if the user has the change permission, it will be all set; otherwise, we need to check more.
            authDel.doUpdateAuth(session, existingSysMeta, Permission.CHANGE_PERMISSION, this.getCurrentNodeId());
```